wagtail/users/templates/wagtailusers/groups/results.html

```diff
@@ -1,12 +1,11 @@
-{% load i18n wagtailadmin_tags %}
+{% extends "wagtailadmin/generic/index_results.html" %}
+{% load i18n %}
 
-{% if groups %}
-    {% component table %}
-
-    {% include "wagtailadmin/shared/pagination_nav.html" with items=page_obj linkurl="wagtailusers_groups:index" %}
-{% elif is_searching %}
-    <p role="alert">{% blocktrans trimmed with query=search_form.q.value %}Sorry, no groups match "{{ query }}"{% endblocktrans %}</p>
-{% else %}
-    {% url 'wagtailusers_groups:add' as wagtailusers_create_group_url %}
-    <p>{% blocktrans trimmed %}There are no groups configured. Why not <a href="{{ wagtailusers_create_group_url }}">add some</a>?{% endblocktrans %}</p>
-{% endif %}
+{% block no_results_message %}
+    {% if is_searching %}
+        <p role="alert">{% blocktrans trimmed with query=search_form.q.value %}Sorry, no groups match "{{ query }}"{% endblocktrans %}</p>
+    {% else %}
+        {% url 'wagtailusers_groups:add' as wagtailusers_create_group_url %}
```

Comment:
How about we reuse `add_url_name` here?

Would be nice to reuse `add_url`, but we'll need to alias it to `wagtailusers_create_group_url` if we don't want to change the translation string, so probably not worth it

Reply:
The latter suggestion is just fine I think - `{% blocktrans trimmed with wagtailusers_create_group_url =add_url %}` should do it. Will give that a go!

Reply:
Updated in 5331a090822d99f46d265bf9d66b948d8ebce977.